src/index.html

```diff
@@ -3,12 +3,63 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin = "anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="moyo_header">
+      <a class="home_link" href="home">
+        <img class="logo_image" src="./images/logo.png"
+        alt="moyoLogo">
```

Comment:
apply code styling recommendations from a [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md) all over the page
```suggestion
        <img 
          class="logo_image" 
          src="./images/logo.png"
          alt="moyo logo"
        >
```